ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -175,6 +175,15 @@ objects:
             wait_expiry_ts="$(($(date +%s) + 90))"
         done
 
+        # wait until the image registry changes propogate to the apiserver to avoid
+        # unnecessary restarts
+        until oc get is -n openshift php 2>/dev/null; do
+          sleep 10
+        done
+        until [[ -n "$( oc get is -n openshift php --template '{{ .status.dockerImageRepository }}' 2>/dev/null )" ]]; do
```

Comment:
Can you drop the earlier `until oc get ...` and just use this loop?  If you need to eat an exit code, maybe:

```sh
until [[ -n "$( oc get is -n openshift php --template '{{ .status.dockerImageRepository }}' 2>/dev/null || true)" ]]; do
```

On the other hand, this code will hopefully not live long, so no need to spend lots of time polishing ;).